diff --git a/node_modules/@graphiql/toolkit/esm/createFetcher.js b/node_modules/@graphiql/toolkit/esm/createFetcher.js
index 6fc3e82..7aecf37 100644
--- a/node_modules/@graphiql/toolkit/esm/createFetcher.js
+++ b/node_modules/@graphiql/toolkit/esm/createFetcher.js
@@ -2,11 +2,11 @@ import { createMultipartFetcher, createSimpleFetcher, isSubscriptionWithName, cr
 export function createGraphiQLFetcher(options) {
     let httpFetch;
     let wsFetcher = null;
-    if (typeof window !== null && window?.fetch) {
+    if (typeof window !== null && window.fetch) {
         httpFetch = window.fetch;
     }
-    if (options?.enableIncrementalDelivery === null ||
-        options.enableIncrementalDelivery !== false) {
+    if (options && (options.enableIncrementalDelivery === null ||
+        options.enableIncrementalDelivery !== false)) {
         options.enableIncrementalDelivery = true;
     }
     if (options.fetch) {
@@ -29,7 +29,7 @@ export function createGraphiQLFetcher(options) {
         if (graphQLParams.operationName === 'IntrospectionQuery') {
             return (options.schemaFetcher || simpleFetcher)(graphQLParams, fetcherOpts);
         }
-        const isSubscription = isSubscriptionWithName(fetcherOpts?.documentAST, graphQLParams.operationName);
+        const isSubscription = isSubscriptionWithName(fetcherOpts && fetcherOpts.documentAST, graphQLParams.operationName);
         if (isSubscription) {
             if (!wsFetcher) {
                 throw Error(`Your GraphiQL createFetcher is not properly configured for websocket subscriptions yet. ${options.subscriptionUrl
diff --git a/node_modules/@graphiql/toolkit/esm/lib.js b/node_modules/@graphiql/toolkit/esm/lib.js
index 2731468..46682ed 100644
--- a/node_modules/@graphiql/toolkit/esm/lib.js
+++ b/node_modules/@graphiql/toolkit/esm/lib.js
@@ -7,7 +7,7 @@ export const isSubscriptionWithName = (document, name) => {
     let isSubscription = false;
     visit(document, {
         OperationDefinition(node) {
-            if (name === node.name?.value) {
+            if (name === (node.name && node.name.value)) {
                 if (node.operation === 'subscription') {
                     isSubscription = true;
                 }
@@ -23,7 +23,7 @@ export const createSimpleFetcher = (options, httpFetch) => async (graphQLParams,
         headers: {
             'content-type': 'application/json',
             ...options.headers,
-            ...fetcherOpts?.headers,
+            ...(fetcherOpts && fetcherOpts.headers),
         },
     });
     return data.json();
@@ -73,7 +73,7 @@ export const createMultipartFetcher = (options, httpFetch) => async function* (g
             'content-type': 'application/json',
             accept: 'application/json, multipart/mixed',
             ...options.headers,
-            ...fetcherOpts?.headers,
+            ...(fetcherOpts && fetcherOpts.headers),
         },
     }).then(response => meros(response, {
         multiple: true,
