diff --git a/node_modules/typedoc-plugin-markdown/dist/utils/front-matter.js b/node_modules/typedoc-plugin-markdown/dist/utils/front-matter.js
index 67dd2fc..4252ffb 100644
--- a/node_modules/typedoc-plugin-markdown/dist/utils/front-matter.js
+++ b/node_modules/typedoc-plugin-markdown/dist/utils/front-matter.js
@@ -34,4 +34,4 @@ ${Object.entries(vars)
     return yaml;
 };
 // prettier-ignore
-const escapeString = (str) => str.replace(/([^\\])"/g, '$1\\"');
+const escapeString = (str) => str.replace(/([^\\])"/g, '$1\\"').replace(/^"/, '\\"');
